src/main/java/org/spongepowered/api/text/format/TextStyles.java

```diff
@@ -79,7 +54,7 @@ private TextStyles() {
      * @return A composite text style from the specified styles
      */
     public static TextStyle of(TextStyle... styles) {
-        return factory.createStyle(styles);
+        return NONE.and(styles);
```

Comment:
:smile: nothing to say here, just dwell in the awesomeness of this for a bit (or it might not be, I don't know)